docs/Makefile

```diff
@@ -1,9 +1,22 @@
 # Minimal makefile for Sphinx documentation
 #
 
+ifeq ($(OS),Windows_NT)
+# It should call the python in your virtualenv.
+# On Windows the name of the python executable inside the virtualenv is
+# generally "python.exe" not python3.exe.
+#
+# And because you generally want to use pyenv to manage multiple versions of
+# Python, you may find that for python3 it actually finds the python3 shim in
+# pyenv. And that shill will not work, because our sphinx is not installed there.
+PYTHON_EXE = python
+else
+PYTHON_EXE = python3
+endif
+
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = python3 -msphinx
+SPHINXBUILD   = $(PYTHON_EXE) -m sphinx
```

Comment:
just change this to  `python` for all platforms 
no need for platform specific stuff then

Reply:
Done.